src/generated/graphql.tsx

```diff
@@ -1748,7 +1748,7 @@ export type UpdateSeatsReservationMutation = { __typename?: 'Mutation', updateSe
 
```

Comment:
Generated UsersQuery is still missing `isExternal` property. Should get fixed by running codegen again 🤔